### PR TITLE
Improve examples: use FQCNs and always add name: to tasks

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
Improve documentation examples of plugins and modules.
1. Always use FQCNs for actions.
2. Always add `name:` to tasks.

This was already the case almost everywhere, but missing in a few cases.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
various

### CODE REVIEW

File: plugins/modules/docker_compose.py [hunk 1]
```diff
@@ -230,10 +232,12 @@
         stopped: true
       register: output
 
-    - ansible.builtin.debug:
+    - name: Show results
+      ansible.builtin.debug:
         var: output
 
-    - ansible.builtin.assert:
+    - name: Verify that web and db service are running
```
Comment (oraNod) [inline plugins/modules/docker_compose.py:239]:
```suggestion
    - name: Verify that web and db services are running
```

File: plugins/modules/docker_compose.py [hunk 2]
```diff
@@ -245,10 +249,12 @@
         restarted: true
       register: output
 
-    - ansible.builtin.debug:
+    - name: Show results
+      ansible.builtin.debug:
         var: output
 
-    - ansible.builtin.assert:
+    - name: Verify that web and db service are running
```
Comment (oraNod) [inline plugins/modules/docker_compose.py:256]:
```suggestion
    - name: Verify that web and db services are running
```

File: plugins/modules/docker_compose.py [hunk 3]
```diff
@@ -257,25 +263,29 @@
   hosts: localhost
   gather_facts: false
   tasks:
-    - community.docker.docker_compose:
+    - name: Scale the web service to 2
```
Comment (oraNod) [inline plugins/modules/docker_compose.py:266]:
```suggestion
    - name: Scale the web service to two instances
```

**Review by oraNod — APPROVED** [review]

**Comment by felixfontein** [comment]
@oraNod thanks!

**Pull request merged**